primitives/src/communities.rs

```diff
@@ -30,7 +32,46 @@ pub type CommunityIndexType = u32;
 pub type LocationIndexType = u32;
 pub type Degree = I64F64;
 pub type NominalIncome = I64F64;
-pub type CommunityIdentifier = H256;
+
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, Default, RuntimeDebug, PartialOrd, Ord)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct CommunityIdentifier {
+    location: [u8; 5],
+    bootstrappers: u32,
```

Comment:
I suggest to call this `digest` as it doesn't matter what went into the digest. Maybe we'd like to include more than just bootstrappers in the future

Reply:
and rather make it a [u8;4], so you have less trouble downstream